src/formats/onenote.ts

```diff
@@ -439,27 +558,26 @@ export class OneNoteImporter extends FormatImporter {
 				this.vault.createFolder(attachmentPath);
 			}
 			catch (e) { }
-			for (let i = 0; i < this.attachmentQueue.length; i++) {
-				const attachment = this.attachmentQueue[i];
+			for (let i = 0; i < attachmentQueue.length; i++) {
+				const attachment = attachmentQueue[i];
 				try {
 					// Every 7 attachments, do a few second break to prevent rate limiting
 					if (i !== 0 && i % 7 === 0) {
 						await new Promise(resolve => setTimeout(resolve, 7500));
 					}
-					const data = (await this.fetchResource(attachment.contentLocation!, 'file')) as ArrayBuffer;
-					await this.app.vault.createBinary(attachmentPath + '/' + attachment.name, data);
+
+					if (!(await this.vault.adapter.exists(`${attachmentPath}/${attachment.name}`))) {
```

Comment:
Should we report the attachment as skipped if it already exists?

Reply:
Yes, that's right